plugin.go

```diff
@@ -62,23 +65,34 @@ type (
 	dispatchFunc func(op PluginOpcode, index int32, value int64, ptr unsafe.Pointer, opt float32) int64
 )
 
-func (d Dispatcher) dispatchFunc(params []*Parameter) dispatchFunc {
+func (d Dispatcher) dispatchFunc(p Plugin) dispatchFunc {
 	return func(op PluginOpcode, index int32, value int64, ptr unsafe.Pointer, opt float32) int64 {
 		switch op {
 		case plugGetParamName:
-			s := (*ascii8)(ptr)
-			copyASCII(s[:], params[index].Name)
+			s := (*ascii32)(ptr)
```

Comment:
I'm not so sure about this change, because VST2.4 [had](https://forum.juce.com/t/eight-character-parameter-names/1206/3) 8 characters limit. I think that some hosts will handle longer strings fine, but it's not aligned with the spec.